packages/cli/src/ui/themes/semantic-tokens.ts

```diff
@@ -55,8 +53,6 @@ export const lightSemanticColors: SemanticColors = {
     focused: lightTheme.AccentBlue,
   },
   ui: {
-    comment: lightTheme.Comment,
```

Comment:
why were these removed?

Reply:
Simplification. I removed 2, `comment` and `symbol`, as they felt redundant on not being used in the code.